resources/translations.mjs

```diff
@@ -0,0 +1,108 @@
+export const todos = {
+    en: [
+        " Oriental Steel Towels",
+        " Bespoke Soft Table",
+        " Ergonomic Fresh Bike",
+        " Luxurious Wooden Cheese",
+        " Gorgeous Fresh Pizza",
+        " Rustic Cotton Shirt",
+        " Incredible Concrete Soap",
+        " Modern Cotton Keyboard",
+        " Oriental Rubber Soap",
+        " Small Bronze Ball",
+        " Awesome Bronze Shoes",
+        " Bespoke Steel Chair",
+        " Awesome Rubber Bike",
+        " Practical Plastic Soap",
+        " Incredible Granite Bacon",
+        " Elegant Metal Keyboard",
+        " Electronic Wooden Sausages",
+        " Tasty Wooden Gloves",
+        " Tasty Concrete Pizza",
+        " Luxurious Metal Cheese",
+        " Awesome Cotton Gloves",
+        " Sleek Soft Car",
+        " Licensed Fresh Salad",
+        " Ergonomic Frozen Towels",
+        " Handmade Plastic Chicken",
+        " Luxurious Cotton Chicken",
+        " Practical Soft Fish",
+        " Oriental Fresh Fish",
+        " Handcrafted Plastic Bacon",
+        " Unbranded Plastic Pants",
+        " Modern Wooden Sausages",
+        " Handmade Steel Shoes",
+        " Rustic Steel Bike",
+        " Gorgeous Frozen Salad",
+        " Handmade Bronze Chicken",
+        " Sleek Granite Bike",
+        " Generic Concrete Sausages",
+        " Incredible Plastic Tuna",
+        " Bespoke Fresh Cheese",
+        " Electronic Fresh Bacon",
+        " Licensed Wooden Car",
+        " Recycled Fresh Fish",
+        " Practical Soft Gloves",
+        " Practical Soft Chips",
+        " Small Soft Chicken",
+        " Intelligent Fresh Mouse",
+        " Tasty Granite Gloves",
+        " Small Steel Shirt",
+        " Refined Concrete Computer",
+        " Sleek Frozen Shirt",
+        " Intelligent Concrete Shoes",
+        " Handmade Rubber Car",
+        " Sleek Cotton Towels",
+        " Unbranded Concrete Hat",
+        " Incredible Plastic Fish",
```

Comment:
Is the space in front of each string intentional?

Reply:
Yeah, these leading whitespace seems unnecessary.

Reply:
removed!